pkg/webui/console/views/device-add/wizard/prompt/index.js

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+
+import Prompt from '@ttn-lw/components/prompt'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+import PropTypes from '@ttn-lw/lib/prop-types'
+import useBeforeUnload from '@ttn-lw/lib/hooks/use-before-unload'
+
+const m = defineMessages({
+  modalTitle: 'Abort end device creation',
+  modalMessage:
+    'Are you sure you want to abort creation of this end device? Unsaved changes in your forms will be lost.',
+  modalApproveText: 'Dismiss and proceed',
```

Comment:
```suggestion
  modalTitle: 'Confirm navigation',
  modalMessage:
    'Are you sure you want to leave this page and abort the end device creation? There are unsaved changes that will be lost.',
  modalApproveText: 'Confirm navigation',
```

Sorry, had to change my mind a little about the message in this particular case.

Additionally, the cancel-message should say `Stay on this page`.